### ISSUE TITLE
Use browsers from new targets config

### ISSUE THREAD

**Issue by mydea** [jeffjewiss/ember-cli-postcss#88]
When using this addon together with e.g. 'autoprefixer', you currently configure autoprefixer like this:

```js
plugins: [
  {
    module: autoprefixer,
    options: {
      browsers: ['last 2 versions', 'ie > 9', 'ios >= 8', '> 5%']
    }
]
```

Following the new possibilities as outlined here: http://rwjblue.com/2017/04/21/ember-cli-targets/, you can now have a `config/targets.js` file like this:

```js
module.exports = {
  browsers: ['> 5%']
};
```

It would be great if we could use that config if no browsers are set directly for autoprefixer (and possibly other plugins).

**Comment by mydea** [comment]
Also see: https://github.com/kimroen/ember-cli-autoprefixer/pull/34

**Comment by jeffjewiss** [comment]
@mydea thanks for submitting this! Yeah, I think this is a great idea. After reading over the article and other repo you mentioned, I think it would make sense to add `project.target.browsers` to each plug-in's base options object as `options.browsers`, which could then be overridden on a per plug-in basis and should follow the formatting expected by a plug-in like autoprefixer.

Does that sounds like what you had in mind? Is there anything I'm missing?

Cheers, thanks!

**Comment by mydea** [comment]
Yes, that sounds perfect!

**Comment by jeffjewiss** [comment]
@mydea would you be able to help me test this? I have this ready on the branch `support-project-targets` with updates to the underlying broccoli plugins. Having another set of eyes on this would be a big help. Thanks!

**Comment by mydea** [comment]
Awesome, I tried it out with the branch, and it seemed to work just fine!

**Comment by jeffjewiss** [comment]
@mydea thanks for the help! I've published this as V3.4.1, which should support using `config/target.js` and overriding the options in compile, filter and on a per plugin basis.

Please open a new issue if there are any bugs with this. Thanks!
